python/tvm/ir/base.py

```diff
@@ -69,6 +69,20 @@ def __init__(self, source_name, line, end_line, column, end_column):
         )
 
 
+@register_object("SequentialSpan")
+class SequentialSpan(Object):
+    """Specifies a location in a source program.
```

Comment:
Please update this doc, since it refers to only a single span. 

Does this class need to be exposed to Python? If it is used only by c++ code, no need to define a python class.